### PR TITLE
Sign up and Login Page working with db/backend skeleton

### PR DESCRIPTION
**Pull request by nidhikay**
- Create Frontend Sign up page  and Login screen and get it hooked up to backend working as well as hooked up to db

Evidence of Sign up working:
SignupPage

Database showing user has been added successfully:
DBnewuser

Evidence of Login working - Happy Path working 
Screen Shot 2022-03-30 at 9 43 09 PM

### CODE REVIEW

File: client/App.jsx [hunk 1]
```diff
@@ -1,8 +1,49 @@
...
 function App() {
+
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const signUp = () => {
+    axios.post('http://localhost:3000/users/signup', {
```
Comment (jonathanhaviv) [inline client/App.jsx]: This should post to localhost:8080 when in dev. The dev-server will proxy to this. And should remove the need for cors.

File: server/routes/users.js [hunk 2]
```diff
@@ -8,7 +8,18 @@ router.get('/', (req, res) => {
 });
 
 router.post('/signup', (req, res, next) => {
-  // db.query(queries.createUser, ['Jonathan', 'Haviv', 'jonathandhaviv@gmail.com', 'jonh', 'password'])
+  const username = req.body.username;
```
Comment (jonathanhaviv) [inline server/routes/users.js]: Would recommend destructuring to make the code cleaner, but not necessary. Also, move this middleware to a userController file.

File: client/App.jsx [hunk 3]
```diff
@@ -1,8 +1,49 @@
...
+      console.log(response);
+    });
+  };
+
   return (
-    <div>Hello Goblin Sharks!</div>
+    <><div>Welcome to MyBookClub!</div>
+      <div className="signup">
+        <h1>Sign Up</h1>
+        <label>Firstname</label>
```
Comment (jonathanhaviv) [inline client/App.jsx:43]: Can you update labels to have the correct htmlFor for each label?

**Review by jonathanhaviv — CHANGES_REQUESTED** [review]
See comments for requested changes.

**Review by jonathanhaviv — APPROVED** [review]

**Review by feiw101 — APPROVED** [review]

**Pull request merged**